lighthouse-core/report/handlebar-helpers.js

```diff
@@ -63,10 +63,9 @@ const handlebarHelpers = {
     return calculateRating(totalScore);
   },
 
-  // Converts an aggregation's score to a rating used for styling.
+  // Converts an aggregation's score to a pass/fail rating.
   getAggregationScoreRating: score => {
-    const rating = calculateRating(score * 100);
-    return rating !== 'good' ? 'poor' : rating; // avg rating maps to failure for aggregation.
+    return score === 1 ? 'good' : 'poor';
```

Comment:
This is strictly for the collapsing behavior. Doesn't affect the score in any way. For "page load performance is fast", the aggregation will close if everything is 100/100 and open if 1-more audits is not 100/100. So if one of the audits is 98/100 and everything else is 100%, we'll still show you were you can do a bit better.